tests/report.test.js

```diff
@@ -126,18 +126,18 @@ describe('Testing report an article ', () => {
       });
   });
 
-  it('Should not allow any user to  get report with data', (done) => {
```

Comment:
Could you please don't remove these tests.

Reply:
why

Reply:
Why are you removing the tests? @kodek-sleuth

Reply:
The test fails on Travis and @niyoceles had raised a bug fix but for some reason the pull request was never in effect

Reply:
> The test fails on Travis and @niyoceles had raised a bug fix but for some reason the pull request was never in effect

@kodek-sleuth, I think you could rebase from develop so that the problem can be solved